.github/workflows/figma_connect_publish.yml

```diff
@@ -0,0 +1,17 @@
+name: Figma Connect Publish on Change
+
+on:
+  push:
+    paths:
+      - packages/react/src/components/**/*.figma.tsx
+    branches:
+      - main
```

Comment:
Do we need to tie this to releases for the package or is it okay to do it on push to `main`? (Or do we need both?)

Reply:
Hmm, releases is problematic if you want to update/fix stuff as we would not do a new release for a figma update. This is why I figured main is best.